lib/ansible/modules/tacp/tacp_network.py

```diff
@@ -0,0 +1,543 @@
+#!/usr/bin/python
+
+# Copyright: (c) 2020, Lenovo
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from ansible.module_utils.basic import AnsibleModule
+from ansible.module_utils.tacp import *
```

Comment:
It's generally a good idea to avoid the star imports .. it pollutes the global namespace, by importing everything, ie also the imports from that module which most probably are not needed.

Can we just `from ansible.module_utils import tacp` then reference `tacp.needed_stuff` ?